internal/pkg/service/common/etcdop/op/atomic.go

```diff
@@ -87,6 +87,15 @@ func (v *AtomicOp) Write(factories ...func() Op) *AtomicOp {
 	return v
 }
 
+func (v *AtomicOp) BeforeWrite(fns ...func() error) *AtomicOp {
+	for _, fn := range fns {
+		v.WriteOrErr(func() (Op, error) {
+			return nil, fn()
+		})
+	}
+	return v
+}
+
```

Comment:
`Write`, alebo `WriteOrErr` moze vratit `op.Op=nil`, v tom pripade callback sluzi iba na nejaku kontrolu alebo pripravu dat, pre nasledujúce callback.

Aby to bolo v kode lepsie citatelne, tak som tu pridal tuto "skratku" `BeforeWrite`